package/package.py

```diff
@@ -37,9 +37,10 @@ class BuildMetadata:
     version_string: str
     buildroot: str
     arch: str
```

Comment:
nit: do we still need it? may be confusing

Reply:
we still use it in some other places